### PR TITLE
Libertine bind mount

### PR DESCRIPTION
**Pull request by paulbendixen**
Added documentation on libertine-container-manager bind mounts so that SD-cards can be accessed

### CODE REVIEW

File: userguide/dailyuse/libertine.rst [hunk 1]
```diff
@@ -116,6 +116,19 @@ To get a shell as user ``phablet`` run::
...
+In order to access your SD-card or any other part of the regular filesystem from inside your libertine container you must create a bind mount. This requires your SD card to be automounted each time you boot the device.
+
+In order to add a bind mount use::
+
+  libertine-container-manager configure -i CONTAINER-IDENTIFIER -b add -p /media/phablet/ID-OF-SD
+  
+You can also make deep links in case you only want parts of your SD-card available in the container.
+In order to use the SD-card as extra space for your container, make sure first to format it using ext4 or similar.
+There is a "feature" in udisk2 that mounts SD-cards that ensures only files ending in .bat, .exe or .com can be executed from the drive if it is (v)fat formatted.
+
```
[the remark this answers is not in the document]
Comment (paulbendixen) [reply userguide/dailyuse/libertine.rst]: Sorry, shouldn't have been qute. IMHO it's not a feature (hence the quotation marks), it's a mis-feature and I've changed it to say as much. I also expanded on what you usually do and why this is not an option on the ubuntu touch.
Please let me know if this is too much

File: userguide/dailyuse/libertine.rst [hunk 2]
```diff
@@ -116,6 +116,19 @@ To get a shell as user ``phablet`` run::
...
+Accessing SD card
+^^^^^^^^^^^^^^^^^
+
+In order to access your SD-card or any other part of the regular filesystem from inside your libertine container you must create a bind mount. This requires your SD card to be automounted each time you boot the device.
+
+In order to add a bind mount use::
+
+  libertine-container-manager configure -i CONTAINER-IDENTIFIER -b add -p /media/phablet/ID-OF-SD
+  
+You can also make deep links in case you only want parts of your SD-card available in the container.
```
[the remark this answers is not in the document]
Comment (paulbendixen) [reply userguide/dailyuse/libertine.rst]: A link to a directory on the SD card in question, added supplemental information and another example

File: userguide/dailyuse/libertine.rst [hunk 3]
```diff
@@ -116,6 +116,19 @@ To get a shell as user ``phablet`` run::
 .. note::
     When you launch bash in this way you will not get any specific feedback to confirm that you are now *inside* the container. You can check ``ls /`` to confirm for yourself that you are inside the container. The listing of ``ls /`` will be different inside and outside of the container.
 
+Accessing SD card
+^^^^^^^^^^^^^^^^^
+
+In order to access your SD-card or any other part of the regular filesystem from inside your libertine container you must create a bind mount. This requires your SD card to be automounted each time you boot the device.
+
```
[the remark this answers is not in the document]
Comment (paulbendixen) [reply userguide/dailyuse/libertine.rst]: I thougt about it some more and I'm not sure it has to be automounted, I now beleive it doesn't even have to be mounted, but I'll have to test that, so for now I've just removed the part about automount

**Comment by doniks** [comment]
Thx!

**Pull request merged**